corehq/apps/app_manager/views/download.py

```diff
@@ -54,13 +56,15 @@ def download_odk_media_profile(request, domain, app_id):
     if not request.app.copy_of:
         username = request.GET.get('username', 'unknown user')
         make_async_build.delay(request.app, username)
+    else:
+        request._always_allow_browser_caching = True
     return HttpResponse(
         request.app.create_profile(is_odk=True, with_media=True),
         content_type="commcare/profile"
     )
 
 
-@safe_download
+@safe_cached_download
```

Comment:
the make new build from mobile feature doesnt use these endpoints, only the profile ones which it seems you did not cache, so that should be fine. however, it does look this will break downloads for all apps that arent builds. i dont know how commonly used those endpoints are, but it looks like if you request any of these files from an app thats not a copy (i.e. is not a build) you will get a cached version instead of the most recent changes. might want to add `if not request.app.copy_of` into your decorator as well